### DISCUSSION TITLE
Add a Redis Version

### DISCUSSION THREAD

**Discussion by jcmcclorey** [leokhoa/laragon#402] [Q&A]
Does anyone know where I can get a different version of Redis to add to Laragon?

**Comment by rescenic** [comment]
I have them in my repo: https://github.com/rescenic/laragon
But normal git only push text, not binaries, when I try git lfs, it seems I need Pro account to host large files.

You can download them here:
https://www.mediafire.com/file/on5k4l66yacnj0t/redis-x64-3.2.100.zip/file
https://www.mediafire.com/file/wy3spiryd9503x8/redis-x64-4.014.2.zip/file
https://www.mediafire.com/file/le9zy0a6xyjxvr2/redis-x64-5.0.14.zip/file
